appveyor.yml

```diff
@@ -1,25 +0,0 @@
-image: Visual Studio 2019
```

Comment:
Can we still keep the file? Even if this line is a simple proxy to `build.ps1`, it's cool to have it for versioning purposes.

Reply:
Of course I'll amend the commit that removes it.

Reply:
@aivascu Sorry if I was not clear enough in my comment. I meant have this file in repository, instead of configuring build script on AppVeyor side via UI. So we use `appveyor.xml` and if we build e.g. old branch - it still works fine.

Was that what you mean by saying `amend`? 😊